src/templates/apps/movement-config.hbs

```diff
@@ -47,14 +43,7 @@
         <label>{{ localize "SFRPG.ActorSheet.Attributes.Speed.MainMovementType" }}</label>
         <div class="form-fields">
             <select name="system.attributes.speed.mainMovement">
-                {{#select system.attributes.speed.mainMovement}}
-                <option value="land">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Land" }}</option>
-                <option value="burrowing">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Burrowing" }}</option>
-                <option value="climbing">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Climbing" }}</option>
-                <option value="flying">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Flying" }}</option>
-                <option value="swimming">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Swimming" }}</option>
-                <option value="special">{{ localize "SFRPG.ActorSheet.Attributes.Speed.Types.Special" }}</option>
-                {{/select}}
+                {{selectOptions (config "speeds") selected=system.attributes.speed.mainMovement}}
```

Comment:
Here you have `(config "speeds")` and in `chassis.hbs` you have `config.speeds` replacing the same list of options, is there a difference between them?

Reply:
The difference is that the `config` variable wasn't available in this template because `src/module/apps/movement-config.js` didn't return it from its `getData()` method. I could easily just add it, but when I was doing this work I was concerned I'd run into exactly this problem several times. Rather than hunt down every `getData()` method that needed to be updated, I decided to register the `config` helper (a better name may be warranted) that would guarantee access to `CONFIG.SFRPG`. At the same time, I didn't bother to replace existing instances of the `config` variable, and I went back and forth a bit on which I preferred.

Configuration is such a static, cross cutting concern that I think the helper should be preferred over passing it as data, but I'm not really adamant about that stance.

Reply:
Ah gotcha, I didn't realize it was a new helper. While I actually like the idea of using a config handlebars helper for that data, I think for this PR we should stick with the `getData()` method of accessing it. This is primarily for the sake of keeping the codebase consistent; unless there's a strong case for using a different method for an individual sheet, it's preferable to use one method across the whole project rather than having two different ways of accessing that data.

If it makes more sense to switch all of the sheets over to using a `config` helper, then I'm definitely not opposed to doing that, we should just transition all the sheets at the same time if we do switch, probably in a separate PR to keep the scope of changes as narrow as it can be. We should also consider, when updating sheet functionality, that Foundry core is transitioning toward their AppV2 framework and phasing out AppV1 slowly. While I'm not that knowledgeable on V2 specifics right now since the deprecation period is long, we should probably start evaluating all changes we make to sheets against the requirements for that framework.